pkg/sql/sem/tree/backup.go

```diff
@@ -212,8 +212,12 @@ func (o *BackupOptions) Format(ctx *FmtCtx) {
 
 	if o.EncryptionPassphrase != nil {
 		maybeAddSep()
-		ctx.WriteString("encryption_passphrase=")
-		o.EncryptionPassphrase.Format(ctx)
+		ctx.WriteString("encryption_passphrase = ")
+		if ctx.flags.HasFlags(FmtShowPasswords) {
```

Comment:
my guess is this is what is tripping up the schedules code, specifically here: https://github.com/cockroachdb/cockroach/blob/master/pkg/ccl/backupccl/create_scheduled_backup.go#L460 

with this change, since that call doesn't pass FmtShowPasswords, it is caught off-guard when tree.AsString doesn't round-trip the AST node back to the the same string

Reply:
Thanks david, that fixes it indeed.